src/Core/Search/Filters/AttributeGroupFilters.php

```diff
@@ -37,6 +37,8 @@ final class AttributeGroupFilters extends Filters
     /** @var string */
     protected $filterId = AttributeGroupGridDefinitionFactory::GRID_ID;
 
+    protected $needsToBePersisted = false;
```

Comment:
There is no reason not to persist this filter, you are simply ignoring the "bug" by adding this Which is not a bug this is the same behavior we have in the hole BO

Reply:
ok @jolelievre, this filter is persisted